config/credentials.yml.enc

```diff
@@ -1 +1 @@
-KM5vdB9P2rMqQAIwEfkAJBgaTmzR0QLPAAy1R4UObjMqwuS3+ab+5oAV7hUm7+WSsbBvIFH/27z+ydaCd3D0TgKOBTa8ZUuquSxXKVXDwDelC5mFpET6CYRTAO2KdePLMX/KJmFqergKkWHxdAp9DxqWY6LslmerTD/lt2PKtfOAg1wJacYMdYxCtPMn2K4MTMshBOMCmLAkfDxqASoSYsslz13QLUrq+36g50qdMKp741efklVGYJqpF9b8OZc3/4ED/jp6KxrZv0GIyLk2maj/ZejzckqeWl95EEFSbrhxcYV/fo5sUnqcGceN2TY18uqwVyyP61z1nXe3DAsNKzwCWVZr9yNxLbGc77hM4gFbp8zCL/vTSLyyUZF3Z/1LwaRhhPST/bputQuHeVHbPb1Qrwt5DxoJRcGFNvjbshBzyppKYcAjqtWX0I5yIqPYh7ZbUQHt--RKaKdGpW8GgZboMg--DBOXE/eCP4sQ+OoXKPVd4A==
\ No newline at end of file
+rAXttXzlfXWjwhTHDRONtMms/0QxzoxEESCdCZQYL402n2F5yc1cgngW1fX8BX4wtJmJnR+piQSWgRVMHVBARhOu7v7YLLiHF81VSemmocAOacHSsuEttja21BCFSBfbP1og7QDKIAQEBfaB4T8XjEAFamfdwJRcyhYV+/8pTRQ8/vFm2AvqEpTSaAHWW1Ltydb5ztLlKUfq8i9rr9awjy6WUs0gfDIPcQ++LyH9BvjEo30tmLRpN/F3mAiQW6n/TBrLvvUGPpfri0F4avg1ZAXl/8VO3VRBjxDtYM4DHfR7OP6fObkMGlyP30fl1MgTZsqcl80iJmpwaDgg3dQPfo1yCPfy3m3byQU1pMrEylEnjz+BDpstc1h0H2m66klORMkmpziQxMteA7i9uHHccDqWtXpyxuIB2GSkPDDgZRNzfOOhhzj6sH2wknsnd75tRVQZgbaFmRJr--FvdRpvk3PCEuuv85--k5ctp6T5DqVw8ykEv7tOvg==
```

Comment:
I think this file shouldn't be here

Reply:
Actually it's safe, since the master.key file is needed to decrypt this one. Will double check the master.key is in the gitignore anyway